testworld/signature_test.go

```diff
@@ -96,11 +219,13 @@ func TestHost_RevokedSigningKey(t *testing.T) {
 
 	res := createDocument(bob.httpExpect, bob.id.String(), typeInvoice, http.StatusOK, defaultInvoicePayload([]string{eve.id.String()}))
 	txID := getTransactionID(t, res)
-	status, message := getTransactionStatusAndMessage(bob.httpExpect, bob.id.String(), txID)
+	status, _ := getTransactionStatusAndMessage(bob.httpExpect, bob.id.String(), txID)
+	//New Client Signature veirifation cause following assert not make sense any more
+	//status is now ok and message is null
```

Comment:
Let's remove this block for now, keep the comment, and assert that status is `success`:
```
// Even though there was a signature validation error, as of now, we keep anchoring document
assert.Equal(t, status, "success")
```

Reply:
ok